submissions/fabunmiTolulope/javascript/conditionals.js

```diff
@@ -0,0 +1,25 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+let number1 = 7;
+let number2 = 123;
+positiveDiff = number2 - number1;
```

Comment:
use let or var here in declaring this variable

Reply:
@adaugochi I have declared the variable using let.